gradle.properties

```diff
@@ -16,12 +16,12 @@ projectDeveloperName=smiley4
 projectDeveloperUrl=https://github.com/SMILEY4
 
 # dependency versions
-versionKtor=2.3.12
+versionKtor=3.0.0
 versionSwaggerUI=5.17.11
 versionSwaggerParser=2.1.22
 versionSchemaKenerator=1.4.3
 versionKotlinLogging=3.0.5
 versionKotest=5.8.0
 versionKotlinTest=1.8.21
```

Comment:
just a suggest, keep the version of `kotlin test` aligned with kotlin

Reply:
got it, bump it to 2.0.20

Reply:
The maintenance of the features of ktor2.+ may continue for some time, and it is recommended to wait until the next major version (for this project) before abandoning ktor 2.+ .